client/my-sites/customer-home/cards/features/domain-upsell/index.jsx

```diff
@@ -54,7 +54,7 @@ export function RenderDomainUpsell() {
 	const domainSuggestionName = domainSuggestion?.domain_name ?? siteSubDomain + '.blog';
 	const domainSuggestionProductSlug = domainSuggestion?.product_slug;
 
-	const searchLink = '/domains/add/' + siteSlug;
+	const searchLink = '/domains/add/' + siteSlug + '?domainAndPlanPackage=true';
```

Comment:
Learned [in my last Calypso PR review](https://github.com/Automattic/wp-calypso/pull/73017/files#diff-e9d5bbfeb0cc388a1ec2501f08f49524a97724605eca06bc1f3f626bf12cfab4R231), Ideally, we should use `addQueryArgs` to create links, so it makes it easier in the future to continue to grow these links or make them dynamically.

Reply:
Thanks for your suggestion, I updated both links (search / purchase) to use `addQueryArgs`